docs/locales/ko_KR/LC_MESSAGES/concepts.po

```diff
@@ -47,7 +47,7 @@ msgid ""
 " use social accounts for log-ins such as Twitter, Facebook, and GitHub."
 msgstr ""
 "클라우드 버전의 Backend.AI API 서비스를 설치하면 개인 장비에 설치하는 것보다 훨씬 더 용이하고 나은 서비스로 이용할 수 있습니다."
-"'cloud.backend.ai <https://cloud.backend.ai>`에서 계정을 만들고 새 API 키페어를 생성하세요."
+"`cloud.backend.ai <https://cloud.backend.ai>`_ 에서 계정을 만들고 새 API 키페어를 생성하세요."
```

Comment:
What's the purpose of the underbar(`_`) in this statement?

Reply:
@lizable Someone translate line 37 same with <https://cloud.backend.ai>`_  . Can I fix without '_' including line 33 ?

Reply:
Underbar(`_`) is used for web links. More details are [here](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#hyperlinks).